README.md

```diff
@@ -72,13 +77,44 @@ class PartyControllerTest < ActionController::TestCase
     with_variant your_ab_test_name: "B" do
       get :show
 
-      assert_rendered "new_show_template_to_be_tested"
-      assert_ab_test_rendered "example"
+      # Optional assertions about page content of the B variant
+    end
+  end
+end
+```
+
+Another common situation is to redirect one variant to another page:
+
+```ruby
+# test/controllers/party_controller_test.rb
+class PartyControllerTest < ActionController::TestCase
+  include GovukAbTesting::MinitestHelpers
+
+  should "redirect the user the B version" do
+    with_variant your_ab_test_name: "B", expected_redirect: { controller: "other_controller", action: "show" } do
```

Comment:
Hmm, wondering if this is putting too much app-specific knowledge into these test helpers. 

What do you think of something like this?

```ruby
with_variant your_ab_test_name: "B", skip_asserts: true do
  assert_response 302
  assert_redirected_to { controller: "other_controller", action: "show" }
end
```